README.md

```diff
@@ -20,3 +20,65 @@ Please check out our [contributing guidelines](CONTRIBUTING.md).
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/teaching-data-science/course-template/binder)
 
 
+
+## Gatsby-Ubuntu Instructions:
```

Comment:
Thanks for that. :clap: 

Can we maybe move this to `CONTRIBUTING.md`?

Reply:
Done!